core-java-modules/core-java-streams-simple/src/main/java/com/baeldung/streams/minmaxbygroup/OrderProcessor.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.streams.minmaxbygroup;
+
+import java.util.DoubleSummaryStatistics;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.Pair;
+
+/** The type Order processor. */
+public class OrderProcessor {
+  /**
+   * Group by category with min max per group.
```

Comment:
let's remove the comments - not necessary

Reply:
Done.